scenarios/01_Elensefar.cfg

```diff
@@ -126,12 +128,12 @@
         [/modify_side]
         [gold]
             side=3
-            {QUANTITY4 amount 0 0 63 84}
+            {QUANTITY4 amount 0 1 63 84}
         [/gold]
     [/event]
 
     [event]
-        {QUANTITY4 name (turn 13) (turn 19) (turn 25) (turn 25)}
+        {QUANTITY4 name (turn 13) (turn 19) (turn 24) (turn 25)}
```

Comment:
This should happen at dawn. It's about the lighthouse getting a good view into the distance, much further than the range that their own light allows them to.

Reply:
ok, so how about `{13 19 25 31}` then?

Reply:
> ok, so how about `{13 19 25 31}` then?

Changed it to this in 3c11649, and also added a comment for future reference.